src/main/java/org/fenixedu/academic/service/services/bolonhaManager/EditCompetenceCourse.java

```diff
@@ -45,11 +45,12 @@ protected void run(String competenceCourseID, String objectives, String program,
     }
 
     protected void run(String competenceCourseID, String name, String nameEn, Boolean basic,
-            CompetenceCourseLevel competenceCourseLevel, CompetenceCourseType type, CurricularStage curricularStage)
+            CompetenceCourseLevel competenceCourseLevel, CompetenceCourseType type, CurricularStage curricularStage, String code)
```

Comment:
This is an API change, the old signature must be kept (deprecated possibly) along with the new one.